### ISSUE TITLE
Chrome autocomplete firing for comboboxes and date pickers

### ISSUE THREAD

**Issue by kklein-celera** [vaadin/flow-components#2510]
### Description

Since we updated to Vaadin 22.0.2, Chrome autocomplete is now firing for ComboBox and DateField components. Chrome is populating the autocomplete suggestions from the saved "organization" values.

### Expected outcome

Autocomplete should be disabled here as it is not helpful.

### Actual outcome

See screen shot below.

![2022-01-12_12-25-56](https://user-images.githubusercontent.com/93628210/149200002-412edc8a-1895-4273-b512-b23bf61eb761.jpg)

### Minimal reproducible example

This seems to affect all ComboBoxes and DateFields where the displayed value is currently blank

### Steps to reproduce

1. Navigate to a view containing a blank ComboBox or DateField component
2. Click on the component to give it focus
3. Chrome autocomplete popup displays with values from saved "organization" names

### Environment

- v22.0.2
- Windows 10
- Chrome 97

### Browsers Affected
- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE 11
- [ ] iOS Safari
- [ ] Android Chrome

**Comment by knoobie** [comment]
Note: In the past (V8) this was fixed / prevented by
 https://github.com/vaadin/framework/blob/cb232f2bffcbd31c64beaa23f1d8dedd9f2ffbac/client/src/main/java/com/vaadin/client/WidgetUtil.java#L1967
